.rubocop.yml

```diff
@@ -52,7 +52,7 @@ Layout/AccessModifierIndentation:
 Layout/ArrayAlignment:
   Enabled: true
 Layout/HashAlignment:
-  Enabled: true
+  Enabled: false
```

Comment:
could you point me to the problem that appeared without changing of this line?

Reply:
![Screenshot from 2020-07-25 10-06-37](https://user-images.githubusercontent.com/10907664/88451393-f06d9480-ce45-11ea-8680-2798506fbd27.png)
---------------------------------------------------------------------------------------------------------------------
![Screenshot from 2020-07-25 10-10-04](https://user-images.githubusercontent.com/10907664/88451401-fa8f9300-ce45-11ea-8cac-054c2aed3529.png)